sys-fs/snapraid/snapraid-12.3-r1.ebuild

```
@@ -0,0 +1,31 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="a backup program for disk array for home media centers"
```

We try to avoid "a ...":
```suggestion
DESCRIPTION="Backup program for disk array for home media centers"
```